ingest/validation/validate_metadata.py

```diff
@@ -211,9 +213,15 @@ def retrieve_ols_term(self, ontology_urls, term, property_name):
         try:
             ontology_shortname, term_id = re.split("[_:]", term)
         except (ValueError, TypeError):
-            raise ValueError(
-                f'{property_name}: Could not parse provided ontology id, "{term}"'
-            )
+            msg = f'{property_name}: Could not parse provided ontology id, "{term}".'
+            if type == "array":
+                if "|" not in term:
+                    msg += (
+                        f" There is only one array value, for ontology id, '{term}.' "
+                        "If unexpected, multiple values must be pipe ('|') delimited."
```

Comment:
what does 'if unexpected' mean here?  

would 'To specify multiple values, make them pipe ('|') delimited' ? be clearer?